src/components/molecules/form-page/Form.tsx

```diff
@@ -1,22 +1,70 @@
-import React from 'react';
+import React, { useState } from 'react';
 import Label from '../../atoms/label/Label';
 import Input from '../../atoms/input/Input';
 import Button from '../../atoms/button/Button';
 
 const Form: React.FC = () => {
+  const [emailValue, setEmailValue] = useState<string>('');
+  const [passwordValue, setPasswordValue] = useState<string>('');
+  const [emailError, setEmailError] = useState<string>('');
+  const [passwordErro, setPasswordErro] = useState<string>('');
```

Comment:
O estado do password erro aqui foge o padrão do texto que foi corrigido no email:
```suggestion
  const [passwordError, setPasswordError] = useState<string>('');
```